### PR TITLE
Changing the `gpt-3.5` model to `gpt-4o-mini`

### PR DESCRIPTION
**Pull request by mrgick**
Hello @deedy5 

There is a model update on the duck.ai .
They added GPT4o-mini instead of gpt-3.5, but gpt-3.5 still works as well now
image

I have added the new model (GPT4o-mini) to your package

Test:
image

Feel free to modify this PR as you wish.
Also, your package is awesome =)

### CODE REVIEW

File: duckduckgo_search/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "6.2.5"
+__version__ = "6.2.6"
```
Comment (deedy5) [inline duckduckgo_search/version.py:1]: Please don't change the version in this PR

**Review by deedy5 — CHANGES_REQUESTED** [review]
Most likely `gpt-4o` is already responding instead of `gpt-3.5`. So just replace `gpt-3.5` with `gpt-4o`

**Review by deedy5 — CHANGES_REQUESTED** [review]
Also use the model name "gpt-4o-mini"

**Comment by deedy5** [comment]
@mrgick In short, change "gpt-3.5" to "gpt-4o-mini" everywhere.

**Comment by mrgick** [comment]
Hello, @deedy5

Revert my last commit and replaced everywhere "gpt-3.5" with "gpt-4o-mini"

My test:
image

Pytest:
image

**Pull request merged**